package/debian/rules

```diff
@@ -18,10 +18,11 @@
 
 override_dh_auto_build:
 	git submodule update --init --recursive
-	make RELEASE=1 K_RELEASE="/usr/lib/kframework" LIBFF_CC=clang-7 LIBFF_CXX=clang++-7 build-node
+	make RELEASE=true deps  -j8
+	make RELEASE=true build -j8 EXTRA_KOMPILE_OPTS='--read-only-kompiled-directory'
 
 override_dh_auto_install:
-	make RELEASE=1 K_RELEASE="/usr/lib/kframework" DESTDIR="${CURDIR}/debian/kevm" INSTALL_PREFIX="/usr" install
+	make DESTDIR="$(CURDIR)/debian/kevm" INSTALL_PREFIX="/usr" install -j8 EXTRA_KOMPILE_OPTS='--read-only-kompiled-directory'
```

Comment:
have you tested that make doesn't rebuild anything if you don't pass RELEASE=true here?

Reply:
Yeah, I double-checked, it doesn't rebuild.

Reply:
Note that `RELEASE` only changes flags, not files generated or anything. So there is no way for `make` to know that the previous invocation did not contain `RELEASE=true` or not.

Reply:
I went ahead and made the `auto_build` and `auto_install` targets more similar anyway, to make it more readable.